Python/Product/VSInterpreters/PackageManager/CondaUtils.cs

```diff
@@ -45,5 +53,73 @@ static class CondaUtils {
             var provider = componentModel.GetService<ICondaLocatorProvider>();
             return provider?.FindLocator()?.CondaExecutablePath;
         }
+
+        /// <summary>
+        /// Determine if an environment is created/managed by conda and thus should be activated.
+        /// </summary>
+        /// <param name="prefixPath">Path to the environment.</param>
+        /// <returns><c>true</c> if it is a conda environment.</returns>
+        internal static bool IsCondaEnvironment(string prefixPath) {
+            return Directory.Exists(Path.Combine(prefixPath, "conda-meta"));
+        }
+
+        /// <summary>
+        /// Activate the base conda environment, capture and return its environment variables
+        /// </summary>
+        /// <param name="condaPath">Path to the base conda environment's conda.exe</param>
+        /// <returns>List of environment variables.</returns>
+        internal async static Task<IEnumerable<KeyValuePair<string, string>>> GetActivatedEnvironmentVariablesAsync(string condaPath) {
```

Comment:
Slightly confusing with the function name and the function below. Change name to be more descriptive to specify that this is the base interpreter?